python-main.js

```diff
@@ -1618,6 +1618,10 @@ function web_editor(config) {
     // Join up the buttons in the user interface with some functions for
     // handling what to do when they're clicked.
     function setupButtons() {
+        if(navigator.platform.match('Win') !== null){
+            $(".roundsymbol").addClass("windowsroundsymbol");
+            $("#small-icons .status-icon").css("line-height","2.2rem");
+        }
```

Comment:
Is this something that could be done in CSS only? I'd rather only have JS doing this kind of thing as a last resort.

Reply:
It doesnt look like I can do this exact thing in CSS.
<img width="555" alt="Screenshot 2019-10-24 11 05 50" src="https://user-images.githubusercontent.com/53710678/67475564-57a7af80-f64e-11e9-9315-6d17e1bb8ad8.png">
I can spend some time looking into a different solution to just changing line height in different browsers if you want?